src/createCycleMiddleware.js

```diff
@@ -1,7 +1,7 @@
 import {run} from '@cycle/xstream-run';
 import xs from 'xstream';
 
-export default function createCycleMiddleware(mainFn, drivers) {
+export default function createCycleMiddleware(mainFn, drivers={}) {
```

Comment:
it's silly but can you add a space for the optionals? `drivers = {}` reads better than `drivers={}`